.github/workflows/release-chart.yaml

```diff
@@ -120,7 +120,7 @@ jobs:
           IFS="${old_ifs}"
 
           if [[ -f "${changes_path}" ]]; then
-            changes="$(cat "${changes_path}")" yq eval --inplace '.annotations["artifacthub.io/changes"] |= strenv(changes)' ./charts/external-dns/Chart.yaml"
+            changes="$(cat "${changes_path}")" yq eval --inplace '.annotations["artifacthub.io/changes"] |= strenv(changes)' ./charts/external-dns/Chart.yaml
```

Comment:
This does not seems legit 
```
$(cat "${changes_path}")
```

There is missing an actual command, example `ls $(cat "changes_path")`

Maybe it suppose to be `cat "${changes_path}" | yq ....`

I've executed it locally <img width="931" alt="Screenshot 2025-06-17 at 19 29 13" src="https://github.com/user-attachments/assets/bc63839b-fca7-448d-8733-ba90b56560bb" />

Reply:
It's legit, `yq` doesn't have support for args so this pattern exposes an inline env variable for it to use as a parameter.

https://mikefarah.gitbook.io/yq/operators/env-variable-operators

Reply:
The defect was the extra double quote, left over from when I converted my multi-chart reusable workflow code to be used here.

Reply:
ok. let's have a look